composer.json

```diff
@@ -75,50 +77,49 @@
             "Utilities\\composer\\DrupalSettings::create"
         ],
         "post-update-cmd": [
-            "DrupalComposer\\DrupalScaffold\\Plugin::scaffold",
```

Comment:
this is no longer required as `drupal/core-composer-scaffold` is now reacts on `composer update`